layouts/django/external.html

```diff
@@ -17,7 +17,6 @@
 {% block govuk_logo %}
   <div class="header-branding">
     <a href="https://www.gov.uk" title="Go to the GOV.UK homepage" class="header-logo">
-      <img src="{% static 'images/crown-logo-black-2x.png' %}" alt="">
```

Comment:
Probably better to leave it but add it to the documentation, then, no?

> On 19 Jan 2016, at 16:41, Tyom Semonov <notifications@github.com> wrote:
> 
> In layouts/django/external.html:
> 
> > @@ -17,7 +17,6 @@
> >  {% block govuk_logo %}
> >    <div class="header-branding">
> >      <a href="https://www.gov.uk" title="Go to the GOV.UK homepage" class="header-logo">
> > -      <img src="{% static 'images/crown-logo-black-2x.png' %}" alt="">
> 
> It can be omitted if it's not required. But since most GOV.UK sites require print stylesheet I think it's a good idea to leave it in the base template. Maybe wrap it in a block to make it optional?
> 
> —
> Reply to this email directly or view it on GitHub.
>